examples/trials/mnist-advisor/mnist.py

```diff
@@ -233,11 +233,7 @@ def generate_default_params():
         # run
         params = generate_default_params()
         params.update(RCV_PARAMS)
-        '''
-        If you use Hyperband, among the hyperparameters (i.e., key-value pairs) received by a trial, 
-        there is one more key called `STEPS` besides the hyperparameters defined by user. 
-        By using this `STEPS`, the trial can control how long it runs.
-        '''
+
         params['batch_num'] = RCV_PARAMS['STEPS'] * 10
```

Comment:
why using "STEPS"? Seems we already rename "STEPS"?

Reply:
This modification is in #911, not modifying the same place in two PR.